lib/oppen/print_stack.rb

```diff
@@ -73,16 +73,17 @@ def output
     #
     # @param token [Token]
     # @param token_width [Integer]
+    # @param delete_last_char [Boolean] whether to delete the last character that was written.
     #
     # @return [Nil]
-    def print(token, token_width)
+    def print(token, token_width, delete_last_char: false)
```

Comment:
why `delete_last_char` and why at this endpoint? It doesn't make a lot of sense from a user POV.

Reply:
We need it at this endpoint because it is the `print` method that determins if a new line is needed or not.